src/commands/get/head.js

```diff
@@ -13,7 +13,11 @@ class HeadCommand extends Command {
   async run() {
     const { flags, args } = this.parse(HeadCommand);
     const { env /* , locale*/ } = args; // package is reserved so we don't try to destructure it
-    const { host, auth } = this.mergeConfig(flags);
+    const { wrhsHost: host, auth } = this.mergeConfig(flags);
+
+    if (!host) {
+      return console.log('Missing warehouse host. Please configure `~/.wrhs` config file, or use the `--host` option.');
```

Comment:
> Please configure `~/.wrhs` config file, or use the `--host` option.

The comma shouldn't be there.  The first part feels awkwardly worded as well. maybe add an `a` in there so it's `Please configure a ~/.wrhs config file`?